backend/src/test/resources/application.yaml

```diff
@@ -0,0 +1,41 @@
+server:
+  port: 8081
+
+hu:
+  flowacademy:
+    worksheet:
+      paging:
+        defaultLimit: 10
+
+spring:
```

Comment:
Szerintem ide elég csak ezt berakni, a többit be tudja húzni az eredeti application.yaml-ből. Kerüljük a duplikációkat.